src/pages/en/user/email/second-factor-verification.html

```diff
@@ -0,0 +1,17 @@
+---
+subject: "${code} is your ${brand} verification code"
+---
+
+<container>
+  <row>
+    <columns small="12">
+      <h1>Verify ${action}</h1>
+      <p><strong>${email}</strong> was used to ${action}. Enter this code to verify your email and ${action}.</p>
```

Comment:
It seems like we need to translate the action as well.  I'm not sure we have a solution for that?  @Astrid-Wire?

Also, I'm pretty sure you can't use action in both those grammatical contexts in all languages and not break at least one or the other.  Maybe just:

```suggestion
      <p><strong>${email}</strong> was used for action ${action}. Enter this code to confirm this in the app.</p>
```